### PR TITLE
Enum repr memory corruption in arrayfire crate

### PR DESCRIPTION
**Pull request by 9prady9**
I have filled in the template to the best of my knowledge.

The following are the URLS for the issue and corresponding PR that fixes the issue on the crate repository.

https://github.com/arrayfire/arrayfire-rust/issues/176
https://github.com/arrayfire/arrayfire-rust/pull/177

Please let me know if any modifications are required.

Thank you.

### CODE REVIEW

File: crates/arrayfire/RUSTSEC-0000-0000.toml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+[advisory]
+# Identifier for the advisory (mandatory). Will be assigned a "RUSTSEC-YYYY-NNNN"
```
Comment (tarcieri) [inline crates/arrayfire/RUSTSEC-0000-0000.toml]: Can you remove the comments please? Thank you!

File: crates/arrayfire/RUSTSEC-0000-0000.toml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+The issue was fixed in crate 3.6.0.
+"""
+
+patched_versions = [">= 3.6.0"]
+
+unaffected_versions = ["<= 3.5.0"]
+
+url = "https://github.com/arrayfire/arrayfire-rust/pull/177"
+
+keywords = ["enum", "repr"]
```
Comment (tarcieri) [inline crates/arrayfire/RUSTSEC-0000-0000.toml]: We've typically added `memory-corruption` to this class of vulnerability.

File: crates/arrayfire/RUSTSEC-0000-0000.toml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+affected_arch = ["x86_64"]
+
+affected_os = ["windows"]
```
Comment (tarcieri) [inline crates/arrayfire/RUSTSEC-0000-0000.toml]: Nice to see this being used! 👍

File: crates/arrayfire/RUSTSEC-0000-0000.toml (hunk 2 above)
Comment (9prady9) [reply crates/arrayfire/RUSTSEC-0000-0000.toml]: I wasn't sure whats the name for memory corruption. Sure will add.

**Pull request merged**